### ISSUE TITLE
Crash in CL151024

### ISSUE THREAD

**Issue by buildermeister** [TwoTwoEleven/MicroManage#7]
Since the last update on the regular branch (CL151024), Satisfactory doesn't start with MicroManage anymore. I get the following error:
`Plugin 'MicroManage' failed to load because module 'MicroManage' could not be loaded.` Very useful...

I found this in the log file, though:

```
[2021.04.20-17.49.04:414][  0]LogWindows: Failed to load 'K:/Steam/steamapps/common/Satisfactory/FactoryGame/Mods/MicroManage/Binaries/Win64/UE4-MicroManage-Win64-Shipping.dll' (GetLastError=127)
[2021.04.20-17.49.04:414][  0]LogModuleManager: Warning: ModuleManager: Unable to load module 'K:/Steam/steamapps/common/Satisfactory/FactoryGame/Mods/MicroManage/Binaries/Win64/UE4-MicroManage-Win64-Shipping.dll' because the file couldn't be loaded by the OS.
[2021.04.20-17.49.05:725][  0]Message dialog closed, result: Ok, title: Message, text: Plugin 'MicroManage' failed to load because module 'MicroManage' could not be loaded.  There may be an operating system error or the module may not be properly set up.
```

I cleared the cache in the Mod Manager and re-downloaded the mods. But that made no difference...

If there is any other information that might help, I will gladly see what I can do.

Not that my russian is any good, but it might be the same error as #6

**Comment by SortingAlgorithmic** [comment]
I am having this issue as well
![image](https://user-images.githubusercontent.com/66043595/115478838-f6e23e80-a214-11eb-91fa-6d40a8348275.png)
![image](https://user-images.githubusercontent.com/66043595/115478849-006ba680-a215-11eb-8a6c-26c99accbbfa.png)

**Comment by Maverik666** [comment]
Since the small update yeterday i have the same Problem:
![image](https://user-images.githubusercontent.com/82930102/115507231-25dfcb00-a27c-11eb-9f08-55666ddce517.png)

**Comment by TwoTwoEleven** [comment]
Fixed in the 0.9.6 release.

**Comment by buildermeister** [comment]
Tnx, you're awesome!!!
